keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1618,6 +1600,32 @@ the number of biindings can become quite large, so let's limit it to
 - xref: [Metrics Provided by OPA Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/docs/metrics/)
 - xref: [Admission Webhook Metrics](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhook-metrics)
 
+### Future Plan
+
+#### Namespace scoped policy binding
+
+**Note**
+The namespace scoped policy binding will require a new API which could not be easily aligned with existing implementation.
```

Comment:
Given that this was previously listed in the KEP, it suggests that the original design didn't meet the design objectives.  It's good that we learned this in alpha instead of beta.  How far off is this design-wise? Let's see a mock up before rushing to a beta that  doesn't meet our goals.